File: T.midterm/JungWonSeok/prob6/prob06.py
Box = [ 1, 2, 3, 1, 2, 3, 5, 6, 5, 6, 10 ]
# Box = [ 1, 2, 3, 1, 2, 3, 5, 6, 5, 6, 10 ]
# Box = [ 9, 8, 11, 2, 3, 5, 45, 21, 32, 11, 45, 45, 8, 13 ]
# Box = [ 7,7,7,7,7,7,7,7 ]



# 이 함수를 문제에서 설명한대로 고치시요
def Tomato( myv ):
    check = sorted(myv, reverse=True) #역순인 확인
    tv =[]
    for q in check: 
        if q not in tv: #만약 중복된게 없으면 추가
            tv.append(q)
    return(tv)



print( Box )

Gong = Tomato( Box )
print( Gong )

File: T.midterm/JungWonSeok/prob6/prob06.cpp
// vector에서 적용할 수 있는 다양한 동작(operator) 중 sorting을 실험해 봄

#include <bits/stdc++.h>
#define allout(MSG,X)   std::cout<<"\n"<<MSG <<" : ";for(auto w:X)std::cout<<w<<", "
using namespace std;

// 여러분은 이 함수를 완성해야 합니다.
vector<int> Tomato( vector<int> myv ){
    vector<int> yourv ;
    yourv = myv ;
    yourv.pop_back() ;
    return ( yourv ) ;

}


int main() {

    vector<int> Box { 1, 2, 3, 1, 2, 3, 5, 6, 5, 6, 10 } ;
//  vector<int> Box { 9, 8, 11, 2, 3, 5, 45, 21, 32, 11, 45, 45, 8, 13 } ;
//  vector<int> Box { 7,7,7,7,7,7,7,7 } ;

    allout(" 작업 전 Box[]", Box ) ;

    auto Apple = Tomato( Box ) ;

    allout(" 작업 후 Apple[]", Apple ) ;


} // end of main ( )
